go.sum

```diff
@@ -0,0 +1 @@
+3644975SUM_LINE
```

Comment:
This file should be removed - some kind of issue copying makefile commands I assume?

Reply:
Done